source/tutorial/triggers-fts.txt

```diff
@@ -0,0 +1,345 @@
+      section of the configuration page and select :guilabel:`+ New Function` 
+      from the drop-down menu. Name the function ``triggers/sendAlerts``.
+      
+      The following JavaScript function returns a message in your application
+      logs when a user enters a task that contains a term stored in the 
+      ``todo.alerts`` collection. 
+
+      The function also includes an Atlas Search query to find other task documents 
+      in the ``todo.Item`` collection that contain the same alert term. The query uses:
+
+      - The :atlas:`$search </atlas-search/query-syntax/>` pipeline stage to query the 
+        collection.
+
+      - The following :atlas:`compound </atlas-search/compound/>` operator clauses:
+
+        - The ``must`` clause and :atlas:`phrase </atlas-search/phrase/>` operator 
+          to query for any tasks with a ``summary`` that contains the alert term.
+        - The ``mustNot`` clause and :atlas:`equals </atlas-search/equals/>` operator to
+          exclude completed tasks.
+
+      - The :manual:`$limit </reference/operator/aggregation/limit>` stage to limit 
+        the output to 5 results.
+      - The :manual:`$project </reference/operator/aggregation/project>` stage to exclude
+        all fields except ``summary``.
+      
+      .. note:: 
+
+         To learn more, see :atlas:`Create and Run Atlas Search Queries 
+         </atlas-search/searching/>`.
+
+      Copy the following code into the function body:
+               
+      .. literalinclude:: /tutorial/backend-examples/sendAlerts.js
+         :language: javascript
+         :caption: functions/triggers/sendAlerts.js
+      
+      When you're finished, click :guilabel:`Save` and deploy the trigger.
+
+   .. step:: Run and Test the App
+
+      Atlas is now set up to alert you when a user creates a task in the app
+      that contains an alert term.
+
+      Rebuild and run the app to make sure everything works. Enter a few 
+      tasks that contain an alert term you've previously entered. Then, 
+      :ref:`view your logs <view-logs>` to see the output from 
+      the trigger. You can filter for your :ref:`trigger logs <logs-trigger>`
+      by selecting the :guilabel:`Triggers` type from the drop-down 
+      menu.
+            
+      For example, if one of your alert terms is ``important``, 
+      the log output for a new task might resemble the following:
+
+      .. code-block:: none
+
+         Logs:
+         [ 
+            "The following task has been added to a to-do list: Complete important tutorial.
+            You've been alerted because it contains the term, important.",
+            "Related incomplete tasks: [
+            {"summary": "Important: Create template app"}, 
+            {"summary": "Add important tasks"},
+            {"summary": "Make sure to read the documentation. This is important."}]"
+         ]
+                  
+What's Next?
+------------
```

Comment:
One other thing we might add - "extend the alert function to call an external service that can send you a text or an email instead of simply logging"